velox/exec/tests/utils/AggregationFuzzer.h

```diff
@@ -16,82 +16,12 @@
 #pragma once
 
 #include "velox/exec/Aggregate.h"
+#include "velox/exec/tests/utils/AggregationFuzzerBase.h"
```

Comment:
I wonder if it makes sense to create exec/fuzzer subdirectory and move fuzzer-related code there. exec/tests could still have AggregationFuzzerTest and AggregationSparkFuzzerTest. And also move input generators and custom verifiers to something liek functions/prestosql/fuzzer. In fact, maybe we could setup something like this:

exec/fuzzer - common fuzzer classes
functions/prestosql/fuzzer - AggregationFuzzerTest + input generators and custom verifiers for Presto functions
functions/sparksql/fuzzer - SparkAggregationFuzzerTest + input generators and custom verifiers for Spark functions